curvefs/src/mds/fs_manager.cpp

```diff
@@ -169,9 +169,14 @@ void FsManager::BackEndFunc() {
     }
 }
 
-FSStatusCode FsManager::CreateFs(const std::string& fsName, FSType fsType,
-                                 uint64_t blockSize, bool enableSumInDir,
-                                 const FsDetail& detail, FsInfo* fsInfo) {
+FSStatusCode FsManager::CreateFs(const ::curvefs::mds::CreateFsRequest* request,
+                                 FsInfo* fsInfo) {
+    std::string fsName = request->fsname();
+    uint64_t blockSize = request->blocksize();
+    FSType fsType = request->fstype();
+    bool enableSumInDir = request->enablesumindir();
+    FsDetail detail = request->fsdetail();
```

Comment:
mark these variables as const reference to avoid unnecessary copy

Reply:
fix